bin/rmt-manual-instance-verify

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'ostruct'
+require 'logger'
+
+options = {}
+parser = OptionParser.new do |opts|
+  opts.banner = "Usage: #{$PROGRAM_NAME} [options]"
+
+  opts.on('-i', '--instance-data INSTANCE_DATA', 'Path to instance data file') do |val|
+    options[:instance_data] = val
+  end
+  opts.on('-m', '--module MODULE', 'Path to instance verification module') do |val|
```

Comment:
Please add ". By default the verification module on the server will be loaded. (Optional)"

Reply:
Done.